package.json

```diff
@@ -43,7 +43,9 @@
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "babel-preset-stage-3": "^6.22.0",
-    "babel-relay-plugin": "https://github.com/alloy/relay/releases/download/v0.9.3/babel-relay-plugin-0.9.3.tgz",
+    "babel-relay-plugin": "0.9.3",
```

Comment:
This isn’t right, it’s extremely important that we use our own fork so that our custom `__id` field works.